tests/test_ditchthecarbs.py

```diff
@@ -11,22 +11,25 @@ def test_host(self):
 
     def test_canonical_url(self):
         self.assertEqual(
-            "https://www.ditchthecarbs.com/easy-keto-hamburger-buns-almond-flour/",
+            "https://thinlicious.com/easy-keto-hamburger-buns-almond-flour/",
```

Comment:
It seems like this entire domain name has been redirected, so.. I wonder if we should rename the scraper and classname too?  (it seems likely, but some more discussion/thought could help)

Reply:
Don't forget the `README.rst` references here too :)